web/src/components/common/GroupBuyMerchantHeader.tsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import styled from 'styled-components';
+
+import {ReactComponent as GPayLogo} from 'assets/gpay-logo.svg';
+
+const HeaderContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const Logo = styled(GPayLogo)`
+  width: 80px;
+`;
+
+const Header = styled.h1`
+  font-size: 32px;
+  font-weight: normal;
+  line-height: 48px;
+  margin: 0;
+`;
```

Comment:
Btw I think this is missing!
```suggestion
const Header = styled.h1`
  font-size: 32px;
  font-weight: normal;
  line-height: 48px;
  margin: 0;
  text-align: center;
`;
```

Reply:
Actually I think it should not be a `1` tag because `h1`s describes the main content of the page, for this the content should be "Sign Up"/"Merchant Sign Up" Instead.
But not sure if it's important, can always change next time also HAHA